### PR TITLE
Update npo_watchlist

### PR DESCRIPTION
**Pull request by mfonville**
* The NPO3 site has been updated to list fragments with the episode too, and list 'articles' also; new trick to only find episodes by looking at running time
* Add running time as npo_runtime (should also help with the date parsing)
* Update the test, added npo_runtime check; but also make sure to test for that things that should NOT be listed are asserted as None

### Motivation for changes:

NPO.nl site has been changed again

### CODE REVIEW

File: flexget/plugins/input/npo_watchlist.py [hunk 1]
```diff
@@ -201,33 +201,36 @@ def _parse_episodes_npo3(self, task, config, series_name, page, series_info):
 
         entries = list()
         for listItem in page.findAll('div', class_='item'):
```
Comment (liiight) [inline flexget/plugins/input/npo_watchlist.py]: I know this wasn't introduced in this PR but please change `listItem` to `list_item`

File: flexget/plugins/input/npo_watchlist.py [hunk 2]
```diff
@@ -267,6 +270,8 @@ def _parse_episodes_std(self, task, config, series_name, page, series_info):
             e['npo_name'] = series_info['npo_name']
             e['npo_description'] = series_info['npo_description']
             e['npo_language'] = series_info['npo_language']
+            if subtitle[1]:  # if runtime is available in the subtext
```
Comment (liiight) [inline flexget/plugins/input/npo_watchlist.py]: Will the previous split always work? If not, this may raise a KeyError

File: flexget/plugins/input/npo_watchlist.py (hunk 1 above)
Comment (mfonville) [inline flexget/plugins/input/npo_watchlist.py]: alright, will do

File: flexget/plugins/input/npo_watchlist.py (hunk 2 above)
Comment (mfonville) [inline flexget/plugins/input/npo_watchlist.py]: The previous split might indeed fail, because there is no easy way to check if 100% of the site confirms to this new subtext format. I will add an extra check

Comment (mfonville) [inline flexget/plugins/input/npo_watchlist.py]: Hmm, I checked the current way, it and this should actually work already correctly... and the split() always returns a list; (even if the splitted symbol is not there);
and the `if`-statement correctly handles the situation if the key exists in the list. It does not spawn any error with any input.

Comment (cvium) [inline flexget/plugins/input/npo_watchlist.py]:
```
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> l = 'nodots'.split('.')
>>> l
['nodots']
>>> l[1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
>>>
```

Comment (mfonville) [inline flexget/plugins/input/npo_watchlist.py]: but it is not `l[1]`

it is `if l[1]:`

Comment (cvium) [inline flexget/plugins/input/npo_watchlist.py]: That will still fail. No matter what precedes `l[1]` it is a direct access to the 2nd element in the list.
```
>>> if l[1]:
...     pass
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
>>>
```

Comment (liiight) [inline flexget/plugins/input/npo_watchlist.py]: It'll still throw that error. Do `if len(l) > 1`

Comment (mfonville) [inline flexget/plugins/input/npo_watchlist.py]: Ok, I will update it
